src/improvements/tasks/MultisigTask.sol

```diff
@@ -254,7 +270,18 @@ abstract contract MultisigTask is Test, Script {
         // TODO change this once we implement task stacking
         nonce = IGnosisSafe(parentMultisig).nonce();
 
-        _setIsNestedSafe();
+        address[] memory owners = IGnosisSafe(parentMultisig).getOwners();
+        {
+            uint256 quorum = IGnosisSafe(parentMultisig).getThreshold();
+            uint256 eoaOwnerCount;
+            for (uint256 i = 0; i < owners.length; i++) {
+                if (owners[i].code.length == 0) {
+                    eoaOwnerCount++;
+                }
+            }
+
+            isNestedSafe = eoaOwnerCount >= quorum ? false : true;
+        }
```

Comment:
Let's retain the old logic here of "if any owner on the safe is an EOA, it is not nested". This definition is a bit arbitrary and doesn't really benefit us